docs/reference/style-guide.mdx

```diff
@@ -0,0 +1,478 @@
+
+### Definition
+
+```
+<Definition term="bastion"/>
+```
+
+<Definition term="bastion"/>
+
+It's worth noting that the `<Definition>` component uses [the glossary page](/reference/glossary).
+If you need to make a new term, add it to the glossary first.
+
+### DismissibleDialog
+
+```
+<DismissibleDialog title="This is a dismissible dialog">
+  This is the content of the dialog.
+</DismissibleDialog>
+```
+
+<DismissibleDialog title="This is a dismissible dialog">
+  This is the content of the dialog.
+</DismissibleDialog>
+
+### DocCardList
+
+```
+<DocCardList items={[
+  {
+    type: 'link',
+    label: 'Terraform Components',
+    description: 'Reusable Terraform "Root" Modules for AWS',
+    href: '/components'
+  },
+  {
+    type: 'link',
+    label: 'Terraform Modules',
+    description: 'Reusable Terraform "Child" Modules',
+    href: '/modules'
+  }
+]}
+/>
+```
+
+<DocCardList items={[
+  {
+    type: 'link',
+    label: 'Terraform Components',
+    description: 'Reusable Terraform "Root" Modules for AWS',
+    href: '/components'
+  },
+  {
+    type: 'link',
+    label: 'Terraform Modules',
+    description: 'Reusable Terraform "Child" Modules',
+    href: '/modules'
+  }
+]}
+/>
+
+It's worth noting that the `<DocCardList>` component can be used without `items` as well. This will render a list of all the pages in the current category using docusaurus built-ins.
+
+### EmbedFile
+
+```
+<EmbedFile filePath="docs/quickstart/handoffs.mdx" />
+```
+
+<EmbedFile filePath="docs/quickstart/handoffs.mdx" />
+
+### Intro
+
+```
+<Intro>
+  This is the content of the intro.
+</Intro>
+```
+
+<Intro>
+  This is the content of the intro.
+</Intro>
+
+### KeyPoints
+
+```
+<KeyPoints title="Example keypoints...">
+- 'Reusable Terraform "Root" Modules for AWS'
+- 'Reusable Terraform "Child" Modules'
+</KeyPoints>
+```
+
+<KeyPoints title="Example keypoints...">
+- 'Reusable Terraform "Root" Modules for AWS'
+- 'Reusable Terraform "Child" Modules'
+</KeyPoints>
+
+### Note
+
+```
+<Note title="optional title">
+  This is the content of the note.
+</Note>
+```
+
+<Note title="optional title">
+  This is the content of the note.
+</Note>
+
+### PillBox
+
+```
+<PillBox>Here's a new topic</PillBox>
+```
+
+<PillBox>Here's a new topic</PillBox>
+
+### Slider
+
+```
+<Slider>
+  <Slide title="Slide 1">
+    <figure>
+      <figcaption>Slide 1 caption</figcaption>
+    </figure>
+  </Slide>
+  <Slide title="Slide 2">
+    <figure>
+      <figcaption>Slide 2 caption</figcaption>
+    </figure>
+  </Slide>
+</Slider>
+```
+
+Slider's can't be demonstrated in a normal page since they take over the table of contents. Therefore, refer to [the jumpstart handoff page](/jumpstart/handoffs) for a working example.
+
+### Steps
+
+```
+<Steps>
+    <Step>
+        #### <StepNumber/> Step 1
+
+        This is the content of step 1.
+    </Step>
+    <Step>
+        #### <StepNumber/> Step 2
+
+        This is the content of step 2.
+    </Step>
+</Steps>
+```
```

Comment:
Also show that steps can be used with ordered and unordered lists, and chapters can be used together with those points.